app/views/KeepOldVrn.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import controllers.routes._
+@import models.Mode
+@import models.requests.DataRequest
+
+@this(
+    layout: views.html.layouts.layout,
+    errorSummary: components.errorSummary,
+    formWithCsrf: FormWithCSRF,
+    yesNoRadio: components.yesNoRadio,
+    button: components.button
+)
+
+@(form: Form[_], mode: Mode)(implicit request: DataRequest[_], messages: Messages, appConfig: FrontendAppConfig)
```

Comment:
What is Mode for? Also, same issue with the previous business name page that was added earlier